modular-kyma/modules.md

```diff
@@ -19,7 +19,7 @@ kyma alpha enable module keda --channel alpha --wait
 
 and `cluster-ip` module using this command:
 ```
-kyma alpha enable module cluster-ip --channel beta --wait
+kyma alpha enable module cluster-ip --channel fast --wait
```

Comment:
To reviewer:

The intention here to let cluster-ip module first deploy in `fast` channel is later can demonstrate the channel switch to `beta` channel for testing more smoothly.